generators/dependencies.js

```diff
@@ -6,6 +6,7 @@ export default {
 	"@sap/approuter": "latest",
 	"@ui5/linter": "latest",
 	"cds-plugin-ui5": "latest",
+    "@types/qunit": "2.5.4",
```

Comment:
`@types/qunit` are also a transitive dependency of the OpenUI5 or SAPUI5 types

Reply:
@nicoschoenteich what do you think? Should we remove them or let it as is and merge?